src/app/auth/auth.component.ts

```diff
@@ -66,14 +61,14 @@ export class AuthComponent implements OnInit, OnDestroy {
             this.errorHandlingService.handleError(error);
           }
           this.authService.changeAuthState(AuthState.NotAuthenticated);
-          this.goToNewGithubLogin();
+          this.goToSessionSelect();
```

Comment:
It looks like `goToSessionSelect` just sets the auth state to `AuthState.NotAuthenticated`.
Should we remove line 63 then?